src/config.rs

```diff
@@ -0,0 +1,67 @@
+use confy;
+use serde::{Serialize, Deserialize};
+use std::io::{self, Write};
+
+const APP_NAME: &str = "lmz";
+const CONFIG_NAME: &str = "config";
+
+#[derive(Default, Debug, Serialize, Deserialize)]
+pub struct Config {
+    pub client_id: String,
+    pub client_secret: String,
+    pub username: String,
+    pub password: String,
+    pub access_token: String,
+}
+
+pub fn path() -> String {
+    let path = confy::get_configuration_file_path(APP_NAME, CONFIG_NAME).expect("expected path");
+    let path_string = String::from(path.to_str().unwrap());
```

Comment:
`Path` and `PathBuf` have a handy [`.display()`](https://doc.rust-lang.org/std/path/struct.Path.html#method.display) method allowing you to produce an `std::path::Display` object implements `std::fmt::Display` so you don't have to mess with things like `Cow<str>`, `OsStr` or `unwrap`s since it handles lossy UTF-8 decoding gracefully (because all rust `str` objects are UTF-8 encoded by default):
```suggestion
    let path_string = format!("{}", path.display());
```